spec/index.bs

```diff
@@ -5520,7 +5520,7 @@ dictionary GPUFragmentState : GPUProgrammableStage {
         Return `true` if all of the following requirements are met:
 
         - |descriptor|.{{GPUFragmentState/targets}}.length must be &le; 8.
-        - For each |colorState| layout descriptor in the list |descriptor|.{{GPUFragmentState/targets}}:
+        - For each non-`null` |colorState| layout descriptor in the list |descriptor|.{{GPUFragmentState/targets}}:
```

Comment:
Something needs to be said about the fact that the sets of targets need to match between the render pass and a pipeline. Here, the wording only happen to care about the non-null attachments, which is insufficient.

Reply:
Addressed myself in another commit